.github/workflows/semantic-release.yml

```
@@ -27,12 +29,30 @@
       contents: write
       issues: write
       pull-requests: write
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)